plugins/inputs/snmp/README.md

```diff
@@ -201,8 +201,42 @@ One [metric][] is created for each row of the SNMP table.
       ## Specifies if the value of given field should be snmptranslated
       ## by default no field values are translated
       # translate = true
+		
+      ## Secondary index table allows to merge data from two tables with
+      ## different index that this filed will be used to join them. There can
+      ## be only one secondary index table.
+      # secondary_index_table = false
+      
+      ## This field is using secondary index, and will be later merged with
+      ## primary index using SecondaryIndexTable. SecondaryIndexTable and
+      ## SecondaryIndexUse are exclusive.
+      # secondary_index_use = false
+      
+      ## Controls if entries from secondary table should be added or not
+      ## if joining index is present or not. I set to true, means that join
+      ## is outer, and index is prepended with "Secondary." for missing values
+      ## to avoid overlaping indexes from both tables. Can be set per field or
+      ## globally with SecondaryIndexTable, global true overrides per field false.
+      # secondary_outer_join = false
 ```
 
+##### Two Table Join
+Snmp plugin can join two snmp tables that have different indexes. For this to work one table
+should have translation field that return index of second table as value. Examples
+of such fields are:
+ * Cisco portTable with translation field: `CISCO-STACK-MIB::portIfIndex`,
+which value is IfIndex from ifTable
+ * Adva entityFacilityTable with translation field: `ADVA-FSPR7-MIB::entityFacilityOneIndex`,
+which value is IfIndex from ifTable
+ * Cisco cpeExtPsePortTable with translation field: `CISCO-POWER-ETHERNET-EXT-MIB::cpeExtPsePortEntPhyIndex`,
+which value is index from entPhysicalTable
+   
```

Comment:
Could you add a configuration example to the docs that uses the new join functionality? I think an example would help beginners and provide a starting point for people to copy-paste and modify. Since you have already mentioned these three examples, maybe base the example on them too?

Is there anything users need to keep in mind when doing two joins? I mean a config that for example joins one table to ifTable and also joins another table to entPhysicalTable.

Reply:
Maybe that is something that could be covered by adding an example config in the testdata folder as explained in #7433

Reply:
Added one example, but it presents both tables before join, and after join. Adding all threes examples would take too much space.